### PR TITLE
Buyside JSON examples

### PR DESCRIPTION
**Pull request by josephpconley**

### CODE REVIEW

File: example/email_action_opened.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (ThreadedLemonAtTraction) [inline example/email_action_opened.json]: @josephpconley general question about these email events... are we not sending over the URL to the market/buyer report?

File: example/marketing_program_member_add.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		"topic": [
+			"marketing"
+		]
+	},
+	"recipient": [
+		"https://bhhs.hsfaffiliates.com/profile/card#me"
+	],
+	"payload": {
+		"type": "MarketingProgramMemberAdd",
+		"agent": {
```
Comment (ThreadedLemonAtTraction) [inline example/marketing_program_member_add.json]: I noticed an Id field in the top level payload of `example/marketing_program_member_remove.json` but I don't see one here. How do we know which report this person is getting added to?

File: example/email_action_opened.json (hunk 1 above)
Comment (josephpconley) [reply example/email_action_opened.json]: @dduran1967 does it make sense to add the url as follows?

```
"messageAttachment": [
      {
        "type": "DigitalDocument",
        "id": "http://stage-valuations-foxroach-com.hvs.buyermls.com/report/#/25354/49461/market-activity-report/430+Foulke+Ln/Springfield/PA/19064"
      }
    ]
```

File: example/marketing_program_member_add.json (hunk 2 above)
Comment (josephpconley) [reply example/marketing_program_member_add.json]: added, thanks!

File: example/email_action_opened.json (hunk 1 above)
Comment (dduran1967) [reply example/email_action_opened.json]: @josephpconley - Yes, that would work if there is an attachment.  You can add it to the example and I'll get it into the schema / docs

File: example/email_action_delivered.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  	"meta": {
+		"topic": [
+			"marketing"
+		]
+	},
+	"type": "DeliveryEvent",
```
Comment (dduran1967) [inline example/email_action_delivered.json]: Are you reporting the email being sent or accepted by the mail server?  In either case, you can use SendAction and the key 'actionStatus' to express a successful (CompletedActionStatus) or unsuccessful (FailedActionStatus) result.

File: example/email_action_delivered.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	"agent": {
+	  "type": "Person",
```
Comment (dduran1967) [inline example/email_action_delivered.json]: is Brian the agent owner of the marketing program?  In reflex, the URI for the agent profile `https://{id}.bhhs.hsfaffiliates.com/profile/card#me` is all you need to include.   If that's not available let's discuss.

File: example/email_action_delivered.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+	  "email": "brian.preston@foxroach.com",
+	"givenName": "Brian",
+	"familyName": "Preston",
+	"identifier": 49461
+	},
+	"object": {
+		"type": "EmailMessage",
+		"id": "45",
```
Comment (dduran1967) [inline example/email_action_delivered.json]: id must be a uri, otherwise you can use identifier.

File: example/email_action_opened.json (hunk 1 above)
Comment (josephpconley) [reply example/email_action_opened.json]: @dduran1967 it's not technically an attachment though, but a link within the message, is that OK?

File: example/email_action_delivered.json (hunk 4 above)
Comment (josephpconley) [reply example/email_action_delivered.json]: @dduran1967 yes he'd be the owner.  I would need to confirm which id to use, do you have an example of a working url?

File: example/email_action_delivered.json (hunk 3 above)
Comment (josephpconley) [reply example/email_action_delivered.json]: Ok will do

File: example/email_action_opened.json (hunk 1 above)
Comment (dduran1967) [reply example/email_action_opened.json]: that's fine, let's discuss on the call

**Pull request merged**